README.md

```diff
@@ -1,6 +1,6 @@
 # dprint-plugin-json
 
-[![](https://img.shields.io/crates/v/dprint-plugin-json.svg)](https://crates.io/crates/dprint-plugin-json) [![CI](https://github.com/dprint/dprint-plugin-json/workflows/CI/badge.svg)](https://github.com/dprint/dprint-plugin-json/actions?query=workflow%3ACI)
+[![crates.io version](https://img.shields.io/crates/v/dprint-plugin-json.svg)](https://crates.io/crates/dprint-plugin-json) [![CI](https://github.com/dprint/dprint-plugin-json/workflows/CI/badge.svg)](https://github.com/dprint/dprint-plugin-json/actions?query=workflow%3ACI)
```

Comment:
This is unrelated to the PR, but I noticed it was bugged (no alt-text) when I opened the file in VSCode.